frontend/src/styles/settings/themeOverride.scss

```diff
@@ -164,3 +164,22 @@
   padding: 0;
   margin: 0;
 }
+// MENU.ITEM
+// Same styling as the default for .ant-dropdown-menu-item,
+// see AuthorizationWrapper.js for the specific use case of this custom class
+.custom-menu-item {
```

Comment:
Are these the exact rules copied from the antd class? I think you can use `@extend` to inherit all of those rules directly instead

Reply:
I also thought that, 

But it did not work, and I was assuming it was due to the class not being mentioned anywhere in our CSS files. Thoughts?